plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -564,7 +564,7 @@ public synchronized List<String> getAllViews(String databaseName)
                 .map(tableName -> getTable(databaseName, tableName))
                 .filter(Optional::isPresent)
                 .map(Optional::get)
-                .filter(table -> TableType.valueOf(table.getTableType()).equals(VIRTUAL_VIEW))
+                .filter(table -> TableType.valueOf(table.getTableType()) == VIRTUAL_VIEW)
```

Comment:
I'd switch this to `table.getTableType().equals(VIRTUAL_VIEW.name())` for safety